### PR TITLE
Us3

### PR DESCRIPTION
**Pull request by sriracha-glitch**
Added Loggers to US3Steps

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 *.class
 *.iml
 out/*
-
-
+*.properties
```
Comment (sriracha-glitch) [inline .gitignore:6]: I didn't want to add my properties file with my credentials so I put it in ignore

File: src/test/resources/features/US3.feature [hunk 2]
```diff
@@ -2,26 +2,12 @@ Feature: Validating all checking accounts page.
   As a User, I want to have the ability to view all my Checking bank account
   so that I can manage my accounts and see account balance and transactions.
 
-  #>As a User,
```
Comment (sriracha-glitch) [inline src/test/resources/features/US3.feature:5]: Removed this user story description and was going to put this in the JIRA ticket

File: src/test/resources/features/US3.feature [hunk 3]
```diff
@@ -2,26 +2,12 @@ Feature: Validating all checking accounts page.
...
-  #- Display header "Date": Date and time when transaction happened.
-  #- Display header "Category": What kind of transaction was made: Income, Misc etc
-  #- Display header "Description": Displays transaction id and type of Transaction
-  #- Display header "Amount": Displays the amount of made transaction
-  #- Display header "Balance": Displays balance amount, after each transaction
-  #3. If User has more than 1 checking accounts, only one of the accounts can be Actived:
-  #- Display ON/OFF toggle button on each Checking account
-  #- Display transaction history for a particular checking account if it is Activated(ON)
+  Background:
+    Given user created an account and is on the "Checking Account" page
```
Comment (sriracha-glitch) [inline src/test/resources/features/US3.feature:6]: Consolidated two given steps in the background

File: .gitignore (hunk 1 above)
Comment (TimurYunusov) [reply .gitignore:6]: sounds good

**Pull request closed without merging**